back/spec/acceptance/users_spec.rb

```diff
@@ -321,13 +321,23 @@
         parameter :group, 'Filter by group_id', required: false
         parameter :can_moderate_project, 'Filter by users (and admins) who can moderate the project (by id)', required: false
         parameter :can_moderate, 'Filter out admins and moderators', required: false
+        parameter :blocked, 'Return only blocked users', required: false
 
         example_request 'List all users' do
           expect(status).to eq 200
           json_response = json_parse(response_body)
           expect(json_response[:data].size).to eq 6
         end
 
+        example_request 'List all users includes user blocking related data' do
+          expect(status).to eq 200
+          json_response = json_parse(response_body)
+          expect(json_response[:data][0][:attributes]).to have_key(:blocked)
+          expect(json_response[:data][0][:attributes]).to have_key(:block_start_at)
+          expect(json_response[:data][0][:attributes]).to have_key(:block_end_at)
+          expect(json_response[:data][0][:attributes]).to have_key(:block_reason)
+        end
+
```

Comment:
@franciclo New test added to confirm user blocking data is returned in regular index action response